lib/cc/services/github_pull_requests.rb

```diff
@@ -45,16 +45,14 @@ def receive_pull_request
     when "pending"
       update_status_pending
     when "success"
-      if config.update_status && config.add_comment
+      if config.update_status || config.add_comment
         update_status_success
         add_comment
-      elsif config.update_status
-        update_status_success
-      elsif config.add_comment
-        add_comment
```

Comment:
My spidey sense is tingling about this change around `config.add_comment`.

Are there any conditions where we want to call `add_comment` and not call `update_status_success` -- /cc @noahd1 @jonathancadepowers